taxcalc/tests/test_compatible_data.py

```diff
@@ -0,0 +1,196 @@
+            '_ID_BenefitSurtax_crt': [0.1],
+            '_II_credit_prt': [0.1],
+            '_II_credit': [[100, 100, 100, 100, 100]],
+            '_CG_brk3': [[1000000, 1000000, 1000000, 1000000, 1000000]],
+            '_ALD_Dependents_Child_c': [1000],
+            '_II_credit_nr': [[1000, 1000, 1000, 1000, 1000]],
+            '_II_credit_nr_prt': [0.1],
+            '_AMT_CG_brk3': [[500000, 500000, 500000, 500000, 500000]],
+            '_AGI_surtax_thd': [[1000000, 1000000, 1000000, 1000000, 1000000]],
+            '_AGI_surtax_trt': [0.5],
+            '_ID_AmountCap_rt': [0.5],
+            '_II_brk7': [[1000000, 1000000, 1000000, 1000000, 1000000]],
+            '_ID_BenefitCap_rt': [0.5],
+            '_DependentCredit_Child_c': [500],
+            '_PT_exclusion_rt': [.2],
+            '_PT_rt7': [.35]
+        }
+    }
+    return _reform_xx
+
+
+@pytest.fixture(scope="module")
+def allparams():
+    """
+    Get current law parameters
+    """
+    return Policy.default_data(metadata=True)
+
+
+@pytest.fixture(scope="module")
+def sorted_param_names(allparams):
+    """
+    Fixture for storing a sorted parameter list
+    """
+    return sorted(list(allparams.keys()))
+
+
+@pytest.fixture(
+    params=[i for i in range(0, int(np.floor(NPARAMS / BATCHSIZE)) + 1)]
+)
+def allparams_batch(request, allparams, sorted_param_names):
+    """
+    Fixture for grouping Tax-Calculator parameters
+    """
+    ix = request.param
+    ix_start = ix * BATCHSIZE
+    ix_end = min((ix + 1) * BATCHSIZE, NPARAMS)
+    pnames = sorted_param_names[ix_start: ix_end]
+    return {pname: allparams[pname] for pname in pnames}
+
+
+@pytest.fixture(params=[True, False], scope="module")
+def tc_objs(request, reform_xx, puf_subsample, cps_subsample):
+    """
+    Fixture for creating TC objects corresponding to using the PUF and using
+    the CPS (only called twice--once for PUF and once for CPS)
+    """
+    puftest = request.param
+    p_xx = Policy()
+    p_xx.implement_reform(reform_xx)
+    if puftest:
+        print('puftest')
+        rec_xx = Records(data=puf_subsample)
+    else:
+        print('cpstest')
+        rec_xx = Records.cps_constructor(data=cps_subsample)
+    c_xx = Calculator(policy=p_xx, records=rec_xx)
+    c_xx.advance_to_year(2018)
+    c_xx.calc_all()
+
+    return p_xx, rec_xx, c_xx, puftest
+
+
+@pytest.mark.requires_pufcsv
+@pytest.mark.pre_release
+def test_compatible_data(cps_subsample, puf_subsample, allparams, reform_xx,
+                         tc_objs, allparams_batch):
+    """
+
+    """
+    # Get taxcalc objects from tc_objs fixture
+    p_xx, rec_xx, c_xx, puftest = tc_objs
+
+    # These parameters are exempt because they are not active under
+    # current law and activating them would deactive other parameters.
+    exempt = ['_CG_ec', '_CG_reinvest_ec_rt']
+
+    # assert len(allparams_i) == 1
+    # pname = list(allparams_i.keys())[0]
+    # param = allparams_i[pname]
```

Comment:
@MattHJensen you may want to remove these commented out lines.

Reply:
Done. Thanks.